source/visionEnhancementProviders/screenCurtain.py

```diff
@@ -300,6 +300,43 @@ def confirmInitWithUser(self) -> bool:
 			return res == wx.YES
 
 
+def isScreenFullyBlack() -> bool:
+	"""
+	Uses wx to check that the screen is currently fully black by taking a screen capture and checking:

```

Comment:
have you considered using the screenBitmap module for this?

Reply:
I've moved the implementation to use that instead, thanks for highlighting this as an option.

Reply:
Having to iterate over this bitmap in python was considerably slower (took 4s on my machine). Using the wx method to calculate the histogram in native code takes 0.4s comparably.